README.md

```diff
@@ -2,391 +1,312 @@
+Now copy docs/quickstart/assessment/webapps/quickstart-1.0.war to the webapps
+directory of a servlet container like Jetty or Tomcat, or use Docker:
 
-**action**<br>
-An optional string used only if integrating with the Data API. Valid options are:
+docker container run --rm -d -v $(pwd)/docs/quickstart/assessment/webapps:/var/lib/jetty/webapps -p 9280:8080 jetty:11.0.7-jdk11
+```
 
-* get
-* set
-* update
-* delete
+Note: your working directory should be the SDK home directory. 
 
-<hr>
+Note: this will load some dependencies, then run a servlet from a JSP file, which uses `App.java` as an adaptor between the servlet and the SDK. 
 
-### Remote
+From this point on, we'll assume that your web server is available at this local address (it will report the port being used when you launch it, by default it's port 9280): 
 
-The Remote class is used to make server side, cross domain requests. Think of it as a cURL wrapper.
+http://localhost:9280/quickstart-1.0/
 
-You'll call either get() or post() with the following arguments:
+The page will load. This is a basic example of an assessment loaded into a web page with Learnosity's assessment player. You can interact with this demo assessment to try out the various Question types.
 
-* [String] URL
-* [Map<String,Object>]  Data payload
+<img width="50%" height="50%" src="docs/images/image-quickstart-examples-assessment.png">
 
-```java
-// Instantiate the SDK Remote class:
-Remote remote = new Remote();
+[(Back to top)](#table-of-contents)
 
-// Instantiate the SDK Remote class with a RequestConfig - https://hc.apache.org/httpcomponents-client-ga/httpclient/apidocs/org/apache/http/client/config/RequestConfig.html:
-RequestConfig requestConfig = RequestConfig.custom().setSocketTimeout(1000).build();
+### **How it works**
+Let's walk through the code for this standalone assessment example. The source files are included under the quickstart folder, in the following locations:
 
-Remote remote = new Remote(requestConfig);
-// Call get() or post() with a URL:
-remote.get("http://schemas.learnosity.com/stable/questions/templates");
+    /learnosity-sdk-java/docs/quickstart/assessment/src/main/java/com/learnosity/quickstart/App.java
+    /learnosity-sdk-java/docs/quickstart/assessment/src/main/webapp/index.jsp
+    /learnosity-sdk-java/docs/quickstart/assessment/src/main/resources/config.properties
 
-// getBody() gives you to body of the request
-String body = response.getBody();
-```
+The first section of code discussed (`App.java`) and is executed server-side. It constructs a set of configuration options for Items API, and securely signs them using the consumer key. The second section is HTML and JavaScript in a JSP page (`index.jsp`) and is executed server-side, and what it generates is loaded in the browser. It renders and runs the assessment functionality.
 
-#### Arguments
+[(Back to top)](#table-of-contents)
 
-**URL**<br>
-A string URL, including schema and path. Eg:
+### **Server-side code**
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 0; Below](https://app.codacy.com/gh/Learnosity/learnosity-sdk-java/pullRequest?prid=8556170)
```suggestion
### **Server-side code**

```